flaky/flaky_pytest_plugin.py

```diff
@@ -269,7 +269,7 @@ def call_runtest_hook(self, item, when, **kwds):
         """
         hookname = "pytest_runtest_" + when
         ihook = getattr(item.ihook, hookname)
-        call_info = CallInfo(
+        call_info = getattr(CallInfo, "from_call", CallInfo)(
```

Comment:
lets add a link to the pr changing the internal api for later comprehension here

Reply:
aka https://github.com/pytest-dev/pytest/pull/4299/